src/ipc/ipc3/handler.c

```diff
@@ -412,7 +412,7 @@ static int ipc_stream_trigger(uint32_t header)
 
 	switch (ipc_command) {
 	case SOF_IPC_STREAM_TRIG_START:
-		cmd = COMP_TRIGGER_START;
+		cmd = COMP_TRIGGER_EARLY_START;
```

Comment:
We should call this PRE_START aligns with PRE_ACTIVE

Reply:
yeah, I thought the names weren't perfect. And I thought about "pre_start" too, but there's already `ssp_pre_start()` at least which I now call from `ssp_early_start()`. I could rename both eventually of course, it's just a static function